<?xml version="1.0"?>
<launch>

 <!-- small description about your node -->
 
 <!--<node pkg="my_package_name" type="my_node_name" name="my_node_name"
 respawn="false" output="screen" args="$(find range_simulation)/config/my_arg_file.yaml"/>-->
 
 <node pkg="range_simulation" type="range_simulation_node" name="RangeSimulationNode" respawn="false" output="screen">
	<rosparam command="load" file="$(find range_simulation)/config/config_range_simulation.yaml"/>
</node>

</launch>
